sw/ground_segment/tmtc/rotorcraft_server.ml

```diff
@@ -174,7 +174,7 @@ let log_and_parse = fun ac_name (a:Aircraft.aircraft) msg values ->
             and desired_alt = foi32value "carrot_up" /. pos_frac in
             a.desired_pos <- Aircraft.add_pos_to_nav_ref nav_ref ~z:desired_alt (desired_east, desired_north);
             a.desired_altitude <- desired_alt +. (hmsl_of_ref nav_ref a.d_hmsl);
-            a.desired_course   <- foi32value "carrot_psi" /. angle_frac
+            a.desired_course   <- fvalue "carrot_psi" /. angle_frac
```

Comment:
Of course not....
But the message are the most important/prominent external interface with a simple API built around it in pprzlink. And at least the important core messages shouldn't be changed light heartedly. Anyone building something on pprzlink (like e.g. FlyingRobotCommander) will have to update it as well if a message that is used changes...

So IMHO "core" messages should not be changed if it doesn't provide a clear advantage or fix something...

Regarding the MD5sum: you will only get the complaint if you rebuilt your aircraft with the new version but didn't upload it... if you only update paparazzi itself (with the changed message) but not rebuild your airframe there is no complaint...

So in my view the "clean" way is to not introduce unnecessary changes to the "external API" (i.e. messages) when there is no additional benefit apart from "internal" refactoring.